code/game/objects/structures/window.dm

```diff
@@ -787,6 +787,12 @@
 /obj/structure/window_frame/update_icon()
 	SSicon_smooth.add_to_queue(src)
 
+/obj/structure/window/full/reinforced/ultra
```

Comment:
remove this definition, not really needed, and definitely for use ONLY by this xanu ship and not any other